src/js/_store/modules/categories.js

```diff
@@ -11,6 +11,8 @@ export default {
     errorMessage: '',
     doneMessage: '',
     loading: false,
+    deleteCategoryId: null,
+    deleteCategoryName: '',
```

Comment:
[imo]

対象が同じ情報を集合として定義したいときは、オブジェクトで定義するのが良いかなと思います！

```suggestion
    deleteCategory: {
      id: null,
      name: '',
    }
```

Reply:
承知いたしました、
オブジェクトの使い方も再確認しようと思います